politeiawww/cmd/politeiawwwcli/client/client.go

```diff
@@ -454,27 +458,89 @@ func (c *Ctx) Logout() error {
 	return err
 }
 
-func (c *Ctx) NewProposal(id *identity.FullIdentity) (*v1.NewProposalReply, error) {
-	payload := []byte("This is a description")
+func (c *Ctx) NewProposal(id *identity.FullIdentity, proposalMarkdown string, attachments []string, random bool) (*v1.NewProposalReply, error) {
+	var np v1.NewProposal
+	var payload []byte
+
+	if random {
```

Comment:
I think it makes sense to move this logic into NewproposalCmd so that you're dealing with the command line flags in the command execute function instead of in the client when possible.